keyboards/chonkerkeys/max/keymaps/default/config.c.template

```diff
@@ -84,14 +84,26 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
         {
             CH_ZOOM_VIDEO_TOGGLE,
             CH_ZOOM_RAISE_HAND_TOGGLE,
-            KC_NO,
-            KC_NO
+            KC_A,
+            KC_MEDIA_NEXT_TRACK
         },
         {
             CH_ZOOM_MUTE_TOGGLE,
             CH_ZOOM_SHARE_SCREEN_START_STOP_TOGGLE,
-            KC_NO,
-            KC_NO
+            CH_SWITCH_WINDOW,
+            KC_MEDIA_PLAY_PAUSE
         }
     }
 };
+
+const uint8_t key_strand[MATRIX_ROWS][MATRIX_COLS] = {
```

Comment:
Is this going to be different per model, or configurable by user? If not, seems it can go to `core.c` directly. The goal is that only user-configurable stuff is in config.c (so that the code-gen code is simpler = less error), please see if this makes sense for you :)

I can add comment/instructions at the top if needed.

Reply:
Yes this is different per model so must be configured by the software / user. This way of configuring is simple and straightforward and if you can make it work in the code-gen, please do. Though in general you won't want, say, the leave meeting button and the strand to be different. You are free to change how we map this, perhaps remove this from config.c, then create a map from keycode to key_strand .

Reply:
Ah, now that I read this again after the thread in basecamp, is this simply converting from QMK's origin to the origin as specified in your diagram in basecamp? If so, it seems we can get rid of it if we decide what origin we should use in the firmware once and for all.